pkg/morph/client/client.go

```diff
@@ -204,6 +205,49 @@ func (c *Client) TestInvoke(contract util.Uint160, method string, args ...any) (
 	return val.Stack, nil
 }
 
+// TestInvokeIterator is the same [Client.TestInvoke] but expands an iterator placed
+// on the stack. Returned values are the values an iterator provides.
+func (c *Client) TestInvokeIterator(contract util.Uint160, method string, args ...any) (res []stackitem.Item, err error) {
+	c.switchLock.RLock()
+	defer c.switchLock.RUnlock()
+
+	if c.inactive {
+		return nil, ErrConnectionLost
+	}
+
+	val, err := c.rpcActor.Call(contract, method, args...)
+	if err != nil {
+		return nil, err
+	}
+
+	if val.State != HaltState {
+		return nil, wrapNeoFSError(&notHaltStateError{state: val.State, exception: val.FaultException})
+	}
+
+	sid, iter, err := unwrap.SessionIterator(val, nil)
+	if err != nil {
+		return nil, fmt.Errorf("iterator expansion: %w", err)
+	}
+
+	const cacheItems = 100
```

Comment:
Use `config.DefaultMaxIteratorResultItems` from NeoGo, server-side constraint is likey to be the same.

Reply:
oh, thanks, i forgot to ask your oppinions, so @roman-khimov, @AnnaShaleva, what value do you think is a good one for fetching containers? 100k containers does not look strange and it would be 1k RPCs for interators as i understand. i took `100` exactly because of its defaultness in `neo-go` repo but wanted to change it after review

Reply:
You'll be limited by the server most of the time, setting 1K won't help a lot. This can be a setting though, if some network has a lot of containers and has some higher settings server-side then you could use a higher setting client-side as well.

Reply:
dont mind using `neo-go`'s default. adding another magic setting isn not worth it yet, IMO